devices/Smart Switch.md

```diff
@@ -68,6 +68,7 @@ Complete Python example is available [here](https://github.com/sinricpro/Python-
 
 > Written with [StackEdit](https://stackedit.io/).
 <!--stackedit_data:
-eyJoaXN0b3J5IjpbLTQwMzcxNTMwMSwyMzAzODc5MjAsNzMwOT
```

Comment:
stackedit_data seems have been chaged. wonder whether it will conflict

Reply:
I edited from my account. As i forked it. 
What does it do? :/

Reply:
Sure. I'll keep it same.

Reply:
Sure i will keep the names same